src/node/esbuildService.ts

```diff
@@ -39,18 +39,21 @@ export function resolveJsxOptions(options: SharedConfig['jsx'] = 'vue') {
 }
 
 // lazy start the service
-let _service: Service | undefined
+let _servicePromise: Promise<Service> | undefined
 
 const ensureService = async () => {
-  if (!_service) {
-    _service = await startService()
```

Comment:
Until this `await`-promise is resolved, subsequent calls to `ensureService` will create additional instances.
But later on, only one of those instances will be stopped.
That's why this commit changes the code to set the variable synchronously.